convert_v2.py

```diff
@@ -0,0 +1,110 @@
+import sys
+import argparse
+import gzip
+import shutil
+import tarfile
+import rarfile
+import patoolib
+import unrar
+
+class FilesFamily:
+
+    def __init__(self, list_of_formats):
+        """
+        This is the constructor function.
```

Comment:
Make sure you set the docstring format to Google, do the following in PyCharm's settings menu 
![image](https://user-images.githubusercontent.com/72790429/95867381-c3a88980-0d71-11eb-9419-f270113612cb.png)

Reply:
done